govulncheck-with-excludes.sh

```diff
@@ -9,7 +9,7 @@ excludeVulns="$(jq -nc '[
 	# fixed in Go 1.20.5+
 	# https://pkg.go.dev/vuln/GO-2023-1840
 	# we already mitigate setuid in our code
-	#"GO-2023-1840", "CVE-2023-29403",
+	"GO-2023-1840", "CVE-2023-29403",
```

Comment:
I have also verified that with this line commented, we get the output for GO-2023-1840 as expected for the latest release binaries.